tests/test_project/admin.py

```diff
@@ -24,9 +25,32 @@ class OperatorInline(admin.StackedInline):
     extra = 0
 
 
-class ProjectAdmin(admin.ModelAdmin):
+class ProjectAdmin(admin.ModelAdmin, UUIDAdmin):
     inlines = [OperatorInline]
     list_display = ['name']
+    readonly_fields = ['uuid']
+    fields = ['name', 'key', 'uuid']
+
+    class Media:
+        js = ('openwisp-utils/js/uuid.js',)
```

Comment:
does it work also without adding `openwisp_utils` to `INSTALLED_APPS`? Can it actually be loaded in the browser correctly? Did you try running the test project in the browser and check this?